### ISSUE TITLE
Question: Multiple schema in multiple database

### ISSUE THREAD

**Issue by skyporter** [influitive/apartment#513]
Does Apartment support switching tenant based on schema AND database ? I want to store many schema on the same database but some specific schema need to be store in their own database.

For exemple DataBase1 store SchemaA and SchemaB, DataBase2 store SchemaC and SchemaD.

**Comment by skyporter** [comment]
Can not anyone answer that question?

**Comment by lcjury** [comment]
The readme says something about this https://github.com/influitive/apartment#tenants-on-different-servers

@skyporter did you achieved what you were trying to do?

**Comment by holamendi** [comment]
@skyporter it's been a while but did you manage to solve this problem? I'm looking into the same situation.

**Comment by skyporter** [comment]
@holamendi nope, I didn't need anymore this requirement. My config still run on the same database.
I just check the main documentation and I think it's possible with this information https://github.com/influitive/apartment#tenants-on-different-servers.
